lib/assets/webpack.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   context: __dirname,
 
   entry: {
-    vendor: ['jquery', 'timeago/jquery.timeago', 'jquery-accessible-autocomplete-list-aria/jquery-accessible-autocomplete-list-aria'],
+    vendor: ['jquery', 'jquery-accessible-autocomplete-list-aria/jquery-accessible-autocomplete-list-aria'],
```

Comment:
Do we want to pull the autocomplete out of here as well? Seems like we should to be consistent with the other dependencies.